pom.xml

```diff
@@ -270,6 +270,13 @@
             <version>1.4.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-all</artifactId>
```

Comment:
As jmock is already used, please continue to use jmock rather than adding another mocking framework.